src/coreclr/tests/runtest.py

```diff
@@ -641,6 +641,8 @@ def call_msbuild(args):
                 "/p:Configuration=%s" % args.build_type,
                 "/p:__LogsDir=%s" % args.logs_dir]
 
+    command += ["/bl:runtest.binlog"]
```

Comment:
If we want to keep this - I think it's a good idea - we should put it in sync with the log naming around line 626, perhaps something like <code>bin_log = log_path + ".binlog"</code>?